types/ckeditor__ckeditor5-engine/src/model/selection.d.ts

```diff
@@ -64,7 +95,7 @@ export default class Selection implements Emitter {
         callback: (info: EventInfo, data: DomEventData) => void,
         options?: { priority?: PriorityString | number | undefined },
     ): void;
-    stopListening(emitter?: Emitter, event?: string, callback?: (info: EventInfo, data: DomEventData) => void): void;
+    stopListening(emitter?: Emitter , event?: string, callback?: (info: EventInfo, data: DomEventData) => void): void;
```

Comment:
Nit pick. Remember to run `prettier` to catch these unneeded changes.

Reply:
Ah, sorry about that. I did run prettier, but it changed a lot of lines that I didn't touch (for instance, all of the double-quotes were changed to single quotes) so I went through and unstaged a bunch of them. It seems I overlooked this in the process.